app.js

```diff
@@ -9,12 +10,19 @@ app.get("/api", getApiDocs)
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
+
 app.all("*",(req, res, next) => {
         res.status(404).send({msg: "Not Found"});
 })
 
 app.use((err, req, res, next) => {
-    console.log(err);
+    if(err.code === '22P02') res.status(400).send({msg: 'Incorrect type for an ID' })
+    else next(err)
+})
+
+app.use((err, req, res , next ) => {
+    res.status(err.status).send({status: err.status, msg: err.msg});
```

Comment:
This final app.use should be for a 500 error - I would look to change this to handle your 500 specifically :)